src/main/java/net/minecraftforge/common/data/ForgeRecipeProvider.java

```diff
@@ -73,10 +73,10 @@ protected void buildRecipes(RecipeOutput consumer) {
         replace(Items.DIAMOND, Tags.Items.GEMS_DIAMOND);
         replace(Items.EMERALD, Tags.Items.GEMS_EMERALD);
         replace(Items.CHEST, Tags.Items.CHESTS_WOODEN);
-        replace(Blocks.COBBLESTONE, Tags.Items.COBBLESTONE_NORMAL);
-        replace(Blocks.COBBLED_DEEPSLATE, Tags.Items.COBBLESTONE_DEEPSLATE);
+        replace(Blocks.COBBLESTONE, Tags.Items.COBBLESTONES_NORMAL);
+        replace(Blocks.COBBLED_DEEPSLATE, Tags.Items.COBBLESTONES_DEEPSLATE);
```

Comment:
Was this change intentional? I got a compile error saying that `COBBLESTONES_NORMAL` and `COBBLESTONES_DEEPSLATE` do not exist. The previous symbols were correct.